roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-controllers.yml.j2

```diff
@@ -28,6 +28,9 @@ spec:
           effect: NoSchedule
         - key: node-role.kubernetes.io/control-plane
           effect: NoSchedule
+{% if policy_controller_extra_tolerations | default(None) %}
```

Comment:
```suggestion
{% if policy_controller_extra_tolerations is defined %}
```

Reply:
Thanks for the code review.
I'm fine with doing the change but the code was taken from existing [coredns-deployment.yml](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/ansible/templates/coredns-deployment.yml.j2#L37-L39). Should I change the code there too or keep the current version in all places ?

Reply:
Doesn't mean existing code is perfect and cannot use a cleanup or modernisation.